tasty-query/shared/src/main/scala/tastyquery/reader/pickles/PickleReader.scala

```diff
@@ -543,7 +545,7 @@ private[pickles] class PickleReader {
       case LITERALclass   => Constant(readTypeRef())
       case LITERALenum =>
         readMaybeExternalSymbolRef() match
-          case sym: Symbol                 => TermRef(NoPrefix, sym)
+          case sym: Symbol                 => TermRef(NoPrefix, sym.asTerm)
```

Comment:
why not?
```suggestion
          case sym: TermSymbol                 => TermRef(NoPrefix, sym)
```

Reply:
Because we'd still need to deal with other `Symbol`s for the match to be exhaustive.

Reply:
id recommend `: @unchecked` but up to you

Reply:
I added an explicit `errorBadSignature` when it's not a `TermSymbol`.